<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="aCardapio">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sprite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>260</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>R$ 3,50</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>340</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fanta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonFanta">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>370</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>340</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>R$ 4,75</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pepsi</string>
      </property>
     </widget>
     <widget class="QLabel" name="tittle">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>My Store</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPepsi">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>230</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>120</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>R$ 6,00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Coca - Cola</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSprite">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>300</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>190</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>R$ 4,00</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewCart">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>440</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Carrinho</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCoke">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>150</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="cPagamento">
     <widget class="QLabel" name="tittle_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pagamento</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonConfirm">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>410</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirmar</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalPricePay">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalPriceValue">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>120</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="totalPricePay_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalPriceValue_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>210</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="totalPricePay_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>251</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Card Number:</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalPriceValue_3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>300</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditCardNumber">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>320</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditName">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBackToCart">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>480</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="dTelaFinal">
     <widget class="QLabel" name="tittle_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>691</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Obrigado por comprar conosco!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBackToCart_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>270</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Novo Pedido</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bCarrinho">
     <widget class="QLabel" name="tittle_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>361</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Itens no Carrinho</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>225</x>
        <y>100</y>
        <width>371</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="totalPriceInCart">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>400</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPay">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>410</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pagar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBackToSelection">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>480</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalPriceInCart_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>420</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
